CCL.Importer/Proxies/CabooseDeviceReplacer.cs

```diff
@@ -53,5 +53,15 @@ public void ReplaceProxies(GameObject prefab)
                 Object.Destroy(antennaProxy);
             }
         }
+
+        public void CacheAndReplaceProxies(GameObject prefab)
+        {
+            return; //Not needed
+        }
+
+        public void MapProxies(GameObject prefab)
+        {
+            return; //Not needed
+        }
```

Comment:
Yes because we can't use default interface implementations.

Alternatively a abstract class could exist which does this, but that seems frivolous for most cases, and that abstract would look like this anyways.